.github/workflows/Bash.yml

```diff
@@ -0,0 +1,69 @@
+# Originally created by Zernit project <https://github.com/RXT0112/Zernit/blob/master/.github/workflows/bash.yml>, granted to kiwibrowser under their preffered license.
+
+name: Bash
+
+# Relevant to events - https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows
+on: 
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
+    paths:
+    - '**.bash'
+    - "tools/dockerfreeze"
+
+jobs:
+  # Linting
+  lint:
+    runs-on: ubuntu-latest
+    container: debian:stable
+    steps:
+      - name: Installing dependencies..
+        run: |
+          # Sync repos
+          # Check for git
+          if ! apt list --installed 2>/dev/null | grep -qP "^git\/stable.*"; then
+            # Check if we can install git
+            if ! apt list | grep -qP "^git\/stable.*"; then
+              apt update
+            elif apt list | grep -qP "^git\/stable.*"; then
+              true
+            else
+              exit 255
+            fi
+            # Install git
+            apt install -y git
+          elif apt list --installed 2>/dev/null | grep -qP "^git\/stable.*"; then
+            true
+          else
+            exit 255
+          fi 
+          # Check for shellcheck
+          if ! apt list --installed 2>/dev/null | grep -qP "^shellcheck\s{1}-.*"; then
+            # Check if we can install shellcheck
+            if ! apt list | grep -qP "^shellcheck\s{1}-.*"; then
+              apt update
+            elif apt list | grep -qP "^shellcheck\s{1}-.*"; then
+              true
+            else
+              exit 255
+            fi
+            # Install shellcheck
+            apt install -y shellcheck
+          elif apt list --installed 2>/dev/null | grep -qP "^shellcheck\s{1}-.*"; then
+            true
+          else
+            exit 255
+          fi 
+      - name: Pulling git dir..
+        uses: actions/checkout@v2
+      - name: Processing files..
+        # Make sure that bash is used
+        shell: bash
+        run: |
+          cd "$GITHUB_WORKSPACE"
+
+          # Process files
+          ## NOTICE: Do not use for loop to avoid pitfall https://mywiki.wooledge.org/BashPitfalls#pf1
+          git --git-dir="$GITHUB_WORKSPACE/.git" ls-files -z -- '*.bash' tools/dockerfreeze | while IFS= read -rd '' file; do
+              printf 'linting bash file %s\n' "$file"
+              shellcheck --external-sources --shell=bash "$file"
+          done
```

Comment:
Missing new line

Reply:
![image](https://user-images.githubusercontent.com/11302521/79700953-9afb7780-8299-11ea-8281-dba52b9f6707.png)

This one? Or which new line do you mean

Reply:
No.... Files that are checked into git get a final newline or diff generation is broken in the last line. 

PS: Please don't send people images of code but instead send links of lines or just copy the code example and mark it somehow. I can't click on your image to land on the line you mean nor can I copy it and find something. I would need to search or type those lines into my search bar if I don't find it immediately.